pilot/pkg/networking/core/listener_waypoint.go

```diff
@@ -483,6 +477,23 @@ func buildWaypointInboundHTTPRouteConfig(lb *ListenerBuilder, svc *model.Service
 	}
 }
 
+// Select the config pertaining to the service being processed.
+func getVirtualServiceForWaypoint(configNamespace string, svc *model.Service, configs []config.Config) *config.Config {
+	for _, cfg := range configs {
+		if cfg.Namespace != configNamespace && cfg.Namespace != svc.Attributes.Namespace {
+			// We only allow routes in the same namespace as the service or in the waypoint's own namespace
```

Comment:
same here for a debug message since the user might want to know this context `"We only allow routes in the same namespace as the service or in the waypoint's own namespace"`, wdyt?

Reply:
This isn't so much that a users config is blocked - we are looking through **all** virtual services for applicable ones. So if we logged it would spam a ton